pkg/kv/kvserver/multi_sst_writer.go

```diff
@@ -64,13 +64,18 @@ type multiSSTWriter struct {
 	rangeDelFrag rangedel.Fragmenter
 }
 
+type multiSSTWriterOptions struct {
+	SSTChunkSize int64
```

Comment:
Add comments? Also, seems like these parameters are not covered with tests.
Make it clear what happens when either/all of these params are zero? See another comment below.

Reply:
Good point, done.